django/contrib/auth/password_validation.py

```diff
@@ -203,11 +202,13 @@ def validate(self, password, user=None):
                     except FieldDoesNotExist:
                         verbose_name = attribute_name
                     raise ValidationError(
-                        _("The password is too similar to the %(verbose_name)s."),
+                        self.get_error_message(verbose_name),
                         code="password_too_similar",
-                        params={"verbose_name": verbose_name},
                     )
 
+    def get_error_message(self, verbose_name):
+        return _("The password is too similar to the %s." % verbose_name)
```

Comment:
I think we should keep the api consistent and have:
```python
                    raise ValidationError(
                        self.get_error_message(),
                        code="password_too_similar",
                        params={"verbose_name": verbose_name},
                    )

    def get_error_message(self):
        return _("The password is too similar to the %(verbose_name)s.")
```

Reply:
OK, I was thinking that might cause an issue when the custom error message doesn't use the `verbose_name` param, but maybe it's fine. I'll update it.